recipes/manim/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "manim" %}
+{% set version = "0.15.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/manim-{{ version }}.tar.gz
+  sha256: ba851ac5e877214c4aae97f28bffe483efe97cb68159480897e13e0b9e9a86a4
+
+build:
+  skip: true  # [py<38]
```

Comment:
If you skip `py<38`, you could also omit the conditional dependencies for it below and make this a `noarch: python` package. Then, we also wouldn't need to skip the Windows build (but can ignore the failing CI). It will become installable on Windows once all missing dependencies are resolved.

Reply:
Reading https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python I think `noarch: python` won't work here as there are scripts (ie `manim` executable) that need to be generated for manim to be run. Not sure if my understanding is correct though.

Reply:
`manim` is defined as an entrypoint below, not a script. So this should work fine.